src/backend/langflow/components/vectorstores/pgvector.py

```diff
@@ -58,17 +56,18 @@ def build(
 
         try:
             if documents is None:
-                return PGVector.from_existing_index(
+                embedding = PGVector.from_existing_index(
```

Comment:
Hey @coolgo0811 

Thanks for this PR! 

I think the name here should be `vectorstore` or something like that. embedding would conflict with the variable that already exists and wouldn't reflect what the object is.

Reply:
Yes! you are right. Thanks for the suggestion.